### ISSUE TITLE
[string.require]p3 Move note's contets

### ISSUE THREAD

**Issue by JohelEGP** [cplusplus/draft#6336]
LWG2994 says
>  There was a request to editorially move the added text in Note 1 into Note 2 after this is approved. 

LWG2994 is added by <https://github.com/cplusplus/draft/pull/6335/commits/27b49130366557adb8e948707e109b198d374b92>.

**Comment by jwakely** [comment]
Yep, I plan to submit a pull request for this after the motions are merged.
